src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Comments = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Comments> = ({ comments }) => (
+
+  (comments.length === 0)
+    ? <b data-cy="NoCommentsMessage">No comments yet</b>
+    : (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />
+        ))}
+      </div>
+    )
 );
```

Comment:
Use return in this case, bc it looks weird
```suggestion
export const CommentList: React.FC<Comments> = ({ comments }) => {
  return !comments.length
    ? <b data-cy="NoCommentsMessage">No comments yet</b>
    : (
      <div className="CommentList">
        {comments.map(comment => (
          <CommentInfo comment={comment} key={comment.id} />
        ))}
      </div>
    )
};
```